runtime/expr/function/ztests/regexp-replace.yaml

```
@@ -1,8 +1,9 @@
 zed: "yield regexp_replace(in, re, new)"
 
 input: |
-  {in:"-ab-axxb-",re:"a(x*)b",new:"T"}
+  {in:"-ab-axxb-",re:"ax*b",new:"T"}
   {in:"-ab-axxb-",re:"a(x*)b",new:"$1"}
+  {in:"-ab-axxb-",re:"a(?P<ZeroOrMoreX>x*)b",new:"foo$ZeroOrMoreX bar"}
```

Use less text so this is more readable.
```suggestion
  {in:"-ab-axxb-",re:"a(?P<X>x*)b",new:"$X"}
```